.eslintrc.js

```diff
@@ -129,8 +129,9 @@ module.exports = {
         'plugin:@typescript-eslint/recommended-requiring-type-checking',
         'plugin:prettier/recommended',
       ],
-      plugins: ['@typescript-eslint', 'jsx-a11y', 'no-only-tests'],
+      plugins: ['@typescript-eslint', 'jsx-a11y', 'no-only-tests', 'blade'],
```

Comment:
Should we call it `@razorpay/blade-eslint`? Just blade seems too confusing

Reply:
`@razorpay/blade-eslint` is not possible.

<img width="706" alt="Screenshot 2023-08-07 at 6 42 51 AM" src="https://github.com/razorpay/blade/assets/46647141/a000986f-6e2d-47c6-8b33-1b19060635c4">


As per the [naming convention](https://eslint.org/docs/latest/extend/plugins#name-a-plugin), we can have `@razorpay-blade-eslint/eslint-plugin`, `@razorpay-blade/eslint-plugin`, `eslint-plugin-razorpay-blade-eslint`, `eslint-plugin-razorpay-blade`.


I've updated it to `@razorpay-blade/eslint-plugin` in the plugin's key we define `@razorpay-blade`.

Reply:
I tried with `@razorpay-blade-eslint/eslint-plugin` and `eslint-plugin-razorpay-blade` names but they doesn't seem to work on CI for some unknown reason.

<img width="908" alt="Screenshot 2023-08-07 at 7 06 39 AM" src="https://github.com/razorpay/blade/assets/46647141/08bd6609-6f23-4f39-902e-e6e405a9b1e6">
<img width="1021" alt="Screenshot 2023-08-07 at 7 19 38 AM" src="https://github.com/razorpay/blade/assets/46647141/3c46b5c6-bda9-4432-9066-85e507c2acbc">

Let's keep `eslint-plugin-blade` only for now. We don't publish this package anywhere. WDYT?

Reply:
Okay. Lets keep it eslint-plugin-blade